test/jobs/split_pdf_job_test.rb

```diff
@@ -6,9 +6,10 @@ class SplitPDFJobTest < ActiveJob::TestCase
   context 'split PDF job' do
     setup do
       Admin.make
-      @assignment = Assignment.make(short_identifier: 'midterm1')
-      f = File.open('db/data/scanned_exams/midterm1.pdf')
-      @exam_template = ExamTemplate.create_with_file(f.read, assignment_id: @assignment.id, filename: 'midterm1.pdf')
+      FileUtils.rm_rf(Dir.glob('data/dev/exam_templates/mdt'))
```

Comment:
Don't do this here. Create the Assignment first, then do a `File.join` with the base path (which should be grabbed from the config file, actually) with the assignment short_identifier.